src/style/shared/colors-light-theme.less

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
This is awesome, but I don't think the world is quite ready. I suggest removing this file from the PR and maintaining it in a separate branch. (But _definitely_ keep it in a separate branch. We might well want to merge it before we ship as a sort of easter egg...)